crates/bevy_render/src/view/visibility/mod.rs

```diff
@@ -157,7 +155,8 @@ pub struct VisibilityBundle {
 }
 
 /// Use this component to opt-out of built-in frustum culling for Mesh entities
-#[derive(Component)]
+#[derive(Component, Reflect, Default)]
```

Comment:
While we're here, could we add `FromReflect` and `ReflectFromReflect` derive and registration?

Reply:
Done. I also found out `Visibility` could do with `FromReflect`.